test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs

```diff
@@ -38,6 +38,8 @@ public class GivenThatIWantToShowHelpForDotnetCommand : TestBase
   nuget            Provides additional NuGet commands.
   msbuild          Runs Microsoft Build Engine (MSBuild).
   vstest           Runs Microsoft Test Execution Command Line Tool.
+  store            Stores the specified assemblies in the runtime package store.
+  help             Show help.
```

Comment:
@mairaw Note that this is a test of the help message. If I change the constant `LocalizableStrings.HelpDefinition` from "Show help" to "Shows help" then it will also affect the other places we use `LocalizableStrings.HelpDefinition`. Fortunately it is only used here (twice)
https://github.com/dotnet/cli/blob/fe49b772ce8e9c66afefbc2b62f43c1b3689a234/src/dotnet/commands/dotnet-help/HelpUsageText.cs

Is it okay that both occurences of "Show help" becomes "Shows help" ? Or should we use separate constants. And will it look strange when the line above (about verbosity) says "Set the verbosity level of the command.".

Reply:
In the other context, it shows next to "Set the verbosity", not "Set*s* the verbosity. There are other tense problems in dotnet help. For example, just above "Provides additional NuGet commands." is "Add/List/Remove" not "Adds/Lists/Removes".

@mairaw, can you look at all of `dotnet help` and file a bug for any changes. This is just showing a string that already shows elsewhere in dotnet help. Once we correct it, it will correct both places.

@mikkelbu you can leave this alone for this PR.

Reply:
Sounds good. I already have an issue assigned to myself in the CLI to do this anyway #5737